README.md

```diff
@@ -73,7 +73,7 @@ To work around the limitations imposed on Terminal windows Spectacle will first
 
 ### Spectacle does not work with all applications
 
-Most applications built with the Cocoa frameworks can be readily manipulated via the _OS X Accessibility Protocol_; allowing Spectacle to interact with nearly every window it encounters. Unfortunately this is not always the case. Spectacle will be unable to manipulate the windows of applications that build their user interfaces in unexpected ways.
+Most applications built with the Cocoa frameworks can be readily manipulated via the _macOS Accessibility Protocol_; allowing Spectacle to interact with nearly every window it encounters. Unfortunately this is not always the case. Spectacle will be unable to manipulate the windows of applications that build their user interfaces in unexpected ways.
```

Comment:
While this change seems in-line with other updates, the developer guide still references **OS X**.  Might minimize confusion to leave this as is for now.